features.json

```diff
@@ -13,7 +13,8 @@
     "ember-routing-transitioning-classes": true,
     "new-computed-syntax": null,
     "ember-testing-checkbox-helpers": null,
-    "ember-metal-stream": null
+    "ember-metal-stream": null,
+    "ember-each-with-index": null
```

Comment:
The general pattern for feature flag names is `<package-name>-<short-feature-name>`.  In this case, I'd suggest `ember-htmlbars-each-with-index`.